contracts/test/NativeTokenSourceTests.t.sol

```diff
@@ -27,7 +27,7 @@ contract NativeTokenSourceTest is NativeTokenBridgeTest, TeleporterTokenSourceTe
     function setUp() public override {
         TeleporterTokenSourceTest.setUp();
 
-        mockWrappedToken = new ExampleWAVAX();
+        mockWrappedToken = new WrappedNativeToken("TOK");
```

Comment:
Is there a reason you named this one `TOK` instead of `AVAX`?

Reply:
no. fixing.